packages/cli/src/ui/themes/theme.ts

```diff
@@ -235,7 +235,7 @@ export function createCustomTheme(customTheme: CustomTheme): Theme {
       customTheme.background?.diff?.added ?? customTheme.DiffAdded ?? '',
     DiffRemoved:
       customTheme.background?.diff?.removed ?? customTheme.DiffRemoved ?? '',
-    Comment: customTheme.ui?.comment ?? customTheme.Comment ?? '',
+    Comment: customTheme.text?.secondary ?? customTheme.Comment ?? '',
```

Comment:
to avoid backsliding, please remove all the non semantic colors now that you have migrated off them everywhere. E.g. Delete all references to AccentRed, etc if that is now possible.

Reply:
I think we still want to keep them since we'll have our base colors (`Colors.AccentRed`) and then those are piped into the semantic colors (`theme.status.error`) that are used in the UI?

Reply:
This is also how Google Material references their colors tokens:

`md.sys.color.primary` => `md.ref.palette.primary40` => `#0B57D0`